whois-nrtm4/src/main/java/net/ripe/db/nrtm4/NrtmFileWriter.java

```diff
@@ -0,0 +1,25 @@
+package net.ripe.db.nrtm4;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import net.ripe.db.whois.common.scheduler.DailyScheduledTask;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class NrtmFileWriter implements DailyScheduledTask {
+
```

Comment:
This is not actually a filewriter, can we rename it something that suggests it's a scheduler?